source/main/ipc/manageAppUpdateChannel.js

```diff
@@ -173,14 +173,10 @@ export const handleManageAppUpdateRequests = (window: BrowserWindow) => {
     // For linux we execute the installer file
     if (environment.isLinux) return installUpdate(filePath);
 
-    // For other OS we launch the installer file
-    const openInstaller: boolean = shell.openItem(filePath);
-    if (!openInstaller)
```

Comment:
@daniloprates shouldn't we have this error case handled?

Reply:
@nikolaglumac this is only for [testing purposes](https://github.com/input-output-hk/daedalus/pull/2195/commits/03ff1637d59f92e6107d86c8f09ed06373ea4552).

Reply:
@nikolaglumac because we use `app.exit(exitCode)` instead of `app.quit()` the `will-quit` event is [not emitted](https://github.com/electron/electron/blob/v8.2.2/docs/api/app.md#appexitexitcode). This could be done only by heavily changing our quitting flow, which I don't think it's worth, since this error will probably never occur, only  if the file is corrupted. We check if the file exists before quitting Daedalus. Since we only rename the file to its final name after the download is completed, if the file is there, there's probably no chance for the error to happen.

Reply:
@nikolaglumac [testing data removed](https://github.com/input-output-hk/daedalus/pull/2195/commits/e50bd82664618d29fe7bb349cfe3ab6538b83a78). Let me know if you want me to go for this change ☝️ but I don't think it's necessary.

Reply:
Sounds good @daniloprates 👍